org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/MethodVerifyTest.java

```diff
@@ -6289,11 +6287,10 @@ public void test077a() {
 	//https://bugs.eclipse.org/bugs/show_bug.cgi?id=128560
 	public void test078() {
 		Map customOptions = getCompilerOptions();
-		customOptions.put(CompilerOptions.OPTION_Compliance, CompilerOptions.VERSION_1_5);
-		customOptions.put(CompilerOptions.OPTION_Source, CompilerOptions.VERSION_1_4);
-		customOptions.put(CompilerOptions.OPTION_TargetPlatform, CompilerOptions.VERSION_1_4);
-		this.runNegativeTest(
-			true,
+		customOptions.put(CompilerOptions.OPTION_Compliance, CompilerOptions.VERSION_9);
+		customOptions.put(CompilerOptions.OPTION_Source, CompilerOptions.getFirstSupportedJavaVersion());
+		customOptions.put(CompilerOptions.OPTION_TargetPlatform, CompilerOptions.getFirstSupportedJavaVersion());
+		this.runConformTest(
```

Comment:
This is testing covariance - so 1.4 and 1.5 are essential. Fine to upgrade, but no need to hardcode version 9. I think you are trying to maintain the +1 nature, that is not needed.

Reply:
OK, will use `CompilerOptions.getLatestVersion()` instead.